ui/widget.js

```diff
@@ -258,6 +259,7 @@ $.Widget.prototype = {
 		this.bindings = $();
 		this.hoverable = $();
 		this.focusable = $();
+		this.classObject = {};
```

Comment:
I still don't like this name. `managedClasses` could work, or whatever else that doesn't use the information free "object".